utils/scripts/nf/__init__.py

```diff
@@ -150,6 +150,15 @@ def docker_registry(self) -> str:
             raise fnfe
         return _docker_registry
         
+    def is_ecr_private_repo(self, _registry_name):
+        """
+        Detect private registry in format <awsaccountid>.dkr.ecr.<awsregion>.amazonaws.com
+        """
+        tmp = _registry_name.split('.')
+        if re.match('[0-9]{12}\.dkr\.ecr\.[a-zA-Z0-9-]{1,}\.amazonaws\.com', _registry_name) is not None:
```

Comment:
reason to test for `None`? It is already "False-like", so returning the `re.match` results in either a "truthy" (match object) or "falsy" (None) value.

Reply:
updated